Topics/User_Experience/navigation design in complex websites.md

```diff
@@ -0,0 +1,64 @@
+## What is Navigation Design ?
+Navigation Design, also referred to as menu design, helps users to easily move through the pages of your website. It utilizes design techniques to allow ease of navigation and provide the user a better sure experience. 
```

Comment:
I believe this is a typo: it should be "better **user** experience", not "sure" experience.

Reply:
great catch ! i'll be sure to make this change!